provisioner/provisioner.go

```diff
@@ -157,7 +157,7 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 }
 
 func (p *Provisioner) Provision(ctx context.Context, ui packer.Ui, comm packer.Communicator, generatedData map[string]interface{}) error {
-	ui.Say("Running mondoo (Version: " + version.Version + ", Build: " + version.Build + ")")
+	ui.Say("Running Mondoo packer provisioner (Version: " + version.Version + ", Build: " + version.Build + ")")
```

Comment:
I am proposing this in place of just `mondoo` as it was slightly confusing with the mondoo client reporting its status directly underneath, e.g.
```
==> amazon-ebs: Running mondoo (Version: v0.2.1, Build: 7d7b5e1)
==> amazon-ebs: Executing Mondoo: <redacted>
    amazon-ebs: → Mondoo 6.3.0 <redacted>
```

Reply:
that's a great improvement, thank you!